hack/auotomate_far_cr.sh

```diff
@@ -0,0 +1,117 @@
+#!/bin/bash
```

Comment:
whatever this thing is doing, I doubt it belongs into this PR?
(also, typo in filename)

Reply:
Correct, it was added here by mistake... Thanks for catching :)